src/other/utils.jl

```diff
@@ -417,3 +417,39 @@ function _findall(B::BitVector)::Union{UnitRange{Int}, Vector{Int}}
     end
     @assert false "should not be reached"
 end
+
+function _drop_metadata!(df::DataFrame)
+    setfield!(df, :metadata, nothing)
+    return nothing
```

Comment:
Returning `df` could make sense here and for other functions below?

Reply:
why? I on purpose wanted the function not to return anything not to risk some strange errors of using it. Note that these functions are internal.

Reply:
That's a relatively common  convention for in-place operations. I thought it could simplify some code, but actually that doesn't seem to be the case so forget it.